tests/fmt/fmt.s

```diff
@@ -10,7 +10,6 @@ TEXT ·Formatting(SB), NOSPLIT, $0
 	ADDQ R8, R8
 
 	// Comment before label.
-label:
```

Comment:
Accidentally removed?

Reply:
Good catch! This is expected behavior since this label is actually unreferenced. I'll add a `JMP` to prevent removal.